lld/COFF/Options.td

```diff
@@ -353,6 +353,12 @@ def fastfail : F<"fastfail">;
 def kernel : F<"kernel">;
 def pdbcompress : F<"pdbcompress">;
 def emitpogophaseinfo : F<"emitpogophaseinfo">;
+defm emittoolversioninfo: B<
+    "emittoolversioninfo",
+    "Emit a tool version info after DOS header (so-called Rich header, default)",
+    "Do not emit a tool version info after DOS header (so-scalled Ricb header)">;
+def novcfeature: F<"novcfeature">;
```

Comment:
To keep these entries alphabetically sorted, move `novcfeature` below `noexp`.

Reply:
I just noticed that theres a typo `so-scalled`, I'll fix that as well